src/pages/Login.js

```diff
@@ -89,6 +90,7 @@ function handleSubmit() {
           { loading ? <TailSpin visible={true}/> : <Button onClick={() => {handleSubmit()}}>
             Login
           </Button> }
+          <Link to="/reset">Change Password</Link>
```

Comment:
I would recommend this get changed to "Forgot Password?" instead :) I feel like that's usually what it says

Reply:
ahh yes your right haha